mining/src/main/java/net/runelite/client/plugins/mining/MiningPlugin.java

```diff
@@ -110,15 +115,26 @@
 	@Inject
 	private MiningConfig config;
 
+	@Inject
```

Comment:
There's no reason to let inject create a instance here.

I would replace this with
`private final GameObjectQuery query = new GameObjectQuery().idEquals(ACTIVE_DAEYALT_ESSENCE);`
this also filters the results with the id, meaning you can just do `activeEssence = query.result(client).first()`